src/containers/Board/WriteFreeBoard/WriteFreeBoard.tsx

```diff
@@ -0,0 +1,73 @@
+import { useRouter } from "next/router";
+import { useQuery, useQueryClient } from "@tanstack/react-query";
+import { useState } from "react";
+
+import * as S from "./emotion";
+import { Quill } from "@/components";
+import { boardsAPI } from "@/apis";
+
+export default function WriteFreeBoard() {
+  const router = useRouter();
+  const { Id } = router.query;
```

Comment:
```suggestion
  const { id } = router.query;
```
파스칼 케이스를 안쓰니까 이게 더 맞지 않을까 싶어요